docs/how-to-guides/running-feast-in-production.md

```diff
@@ -2,11 +2,29 @@
 
 ## Overview
 
+After learning about Feast concepts and playing with Feast locally, you're now ready to use Feast in production.
+This guide aimed to help with the transition from a sandbox project to production-grade deployment in the cloud or on-premise. 
+
+Overview of typical production configuration is given below:
+
+![Overview](production-simple.png)
+
+{% hint style="success" %}
+**Important note:** Feast is flexible and doesn't require you to install unneeded parts.
+For example, you might not have a stream source and, thus, no need to write features in real-time to Online Store. 
+Or you might not need retrieving online features. Feast is ready to adjust to your needs.
```

Comment:
Can this sentence be made more clear?

Also, what does "Feast is ready to adjust to your needs" mean?

Reply:
> Can this sentence be made more clear?
which one?